components/DatasetsTable/DatasetsToolbar.tsx

```diff
@@ -0,0 +1,30 @@
+import type { ReactNode } from 'react';
+
+import { css } from '@emotion/react';
+import { Grid, useTheme } from '@material-ui/core';
+
+export interface DatasetsToolbarProps {
+  /**
+   * Elements to be wrapped by the component.
+   */
+  children: ReactNode;
+}
+
+/**
+ * Wraps elements in the DataTable toolbar and adds spacing between them.
+ */
+export const DatasetsToolbar = ({ children }: DatasetsToolbarProps) => {
+  const theme = useTheme();
+  return (
+    <Grid
+      container
+      alignItems="center"
+      css={css`
+        gap: ${theme.spacing()}px;
```

Comment:
Any reason to not use the spacing property of a Grid here? Also I think "row" is the default direction.

Reply:
I think the usage of Grid can be confusing here without the Grid items. I'd switch to wrapping grid items with spacing prop. If you want to reduce the complexity in the parent scope, you could map over the passed children prop and wrap each child in a grid item.